src/rtc.h

```diff
@@ -42,6 +42,7 @@
 #include "stm32_def.h"
 #include "backup.h"
 #include "clock.h"
+#include <string.h>
```

Comment:
As only needed for the memcpy for the `RTC_StoreDate` in case of F1 it should be only on rtc.c.

Reply:
moved to `rtc.c`